apihub/subscription/models.py

```diff
@@ -15,19 +67,27 @@ class Subscription(Base):
     __tablename__ = "subscriptions"
 
     id = Column(Integer, primary_key=True, index=True)
-    application = Column(String, unique=True, index=True, nullable=False)
+    # duplicate tier to make it easier to query.
```

Comment:
comment is not needed here. We don't see it is a duplicate

Reply:
ok